tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -168,3 +168,25 @@ def test_java_default(self):
                          desc='     the concatenated string\n')]]
 
         self.assertEqual(expected, parsed_docs)
+
+
+class AssembledDocumentationTest(unittest.TestCase):
+
+    def test_python_assembly(self):
```

Comment:
I think I need to persist a bit on this, as when I read the class and test-function name, I think someone who has never seen this code doesn't understand what is actually tested^^

1. You test the function `from_metadata`, so I would call the test function `test_from_metadata`
2. `from_metadata` is language independent, so the test imo belongs to the general class, even though you use the python-default docstyle-definition.
3. It's some kind of "assembly" (assembling the parsed metadata), but it does not really refer to the function name. So the class should be called at least `FromMetadataTest` (or `MetadataAssemblyTest`)